meldingen_core/models.py

```diff
@@ -22,3 +22,16 @@ class User:
 
     username: str
     email: str
+
+
+@dataclass
+class Question:
+    text: str
+    classification: Classification
```

Comment:
Not sure if we need to keep the order of the questions 🤔 could be handy when displaying the questions/answers when a form has changed or even has been deleted?

Reply:
As discussed, I'll remove this relation for now in favor of introducing a form model that will indirectly provide this relationship.